activesupport/test/cache/behaviors/cache_store_behavior.rb

```diff
@@ -406,7 +406,7 @@ def test_hash_as_cache_key
   end
 
   def test_keys_are_case_sensitive
-    key = SecureRandom.alphanumeric
+    key = SecureRandom.alphanumeric.downcase
     @cache.write(key, "bar")
     assert_nil @cache.read(key.upcase)
   end
```

Comment:
I wonder if another option would be to not use `SecureRandom.alphanumeric` at all

I think we can test the case sensitivity without using any random behaviour and by relying on an explicit key instead 🤔

Reply:
Hmm, you're right 😄. I've made the changes accordingly. Thanks!